src/Assessment.js

```diff
@@ -24,6 +25,9 @@ export default class Assessment extends Component {
       return null;
     }
     const title = assessmentNode.getAttribute("title");
+    const isFromQuizzesNext = !!assessmentNode.getAttribute(
```

Comment:
Suggestion
maybe we should use a single phrase for new quiz?
'quizzesnext' or 'newquizzes'

Reply:
`quizzesnext` phrase is the technical contract at all system components.
`New Quizzes` is the user facing name.

Reply:
I mean the variable names
here it's `isFromQuizzesNext` and in `src/AssessmentListItem.js:42` it's `isFromNewQuizzes`